coalib/processes/Processing.py

```diff
@@ -385,8 +396,16 @@ def instantiate_processes(section,
     # the whole project is accessible to every bear. However, local bears are
     # run only for the changed files if caching is enabled.
 
+    if len(use_raw_files) > 1:
+        log_printer.err("Bears that uses raw files can't be mixed with "
+                        'Bears that uses text files. Skipping this '
+                        'section.')
+        mixed = True
+    use_raw_files = use_raw_files.pop() if len(use_raw_files) != 0 else False
```

Comment:
Actually, it expects it to be only one object inside the Set. If there's more than one, this section will be skipped anyway since it's mixing bears that uses raw files and the ones that doesn't.

the `if len(use_raw_files) != 0` is added because there are some tests that executes this function without giving a bear.